src/Symfony/Component/Serializer/Normalizer/DateTimeNormalizer.php

```diff
@@ -59,12 +61,16 @@ public function getSupportedTypes(?string $format): array
     /**
      * @throws InvalidArgumentException
      */
-    public function normalize(mixed $object, string $format = null, array $context = []): string
+    public function normalize(mixed $object, string $format = null, array $context = []): int|float|string
     {
         if (!$object instanceof \DateTimeInterface) {
             throw new InvalidArgumentException('The object must implement the "\DateTimeInterface".');
         }
 
+        if (!isset($this->defaultContext[self::TIMESTAMP_CAST_KEY])) {
+            trigger_deprecation('symfony/serializer', '7.1', 'Not setting the "%s" in defaultContext for "%s" is deprecated. It will default to "true" in 8.0.', self::TIMESTAMP_CAST_KEY, self::class);
+        }
```

Comment:
This was requested by @mtarld, so not sure then :)

Reply:
Indeed, as I've written in this comment https://github.com/symfony/symfony/pull/53056#discussion_r1426664940, I think it could be great to cast timestamps by default in 8.0. So, we need to set the default context value to `true` in 8.0 (and therefore it changes the 7.x default behavior)

Reply:
I've probably targeted wrong version for recipe?